ui/src/app/pages/login/account-creation/account-creation-survey.tsx

```diff
@@ -131,30 +135,31 @@ export class AccountCreationSurvey extends React.Component<AccountCreationSurvey
       <Section header='Race'>
         <FlexColumn style={styles.checkboxAreaContainer}>
           {AccountCreationOptions.race.map((race) => {
-            return this.createOptionCheckbox(race.label, 'race', race.value, race.value.toString());
+            return this.createOptionCheckbox('race', race);
           })}
         </FlexColumn>
       </Section>
 
       {/*Ethnicity section*/}
-      <DropDownSection header='Ethnicity' options={AccountCreationOptions.ethnicity}
+      <DropDownSection data-test-id='dropdown-ethnicity'
+                       header='Ethnicity' options={AccountCreationOptions.ethnicity}
                        value={demographicSurvey.ethnicity}
                        onChange={(e) => this.updateDemographicAttribute('ethnicity', e)}/>
       <Section header='Do you identify as lesbian, gay, bisexual, transgender, queer (LGBTQ),
 or another sexual and/or gender minority?'>
         <FlexColumn>
           <FlexRow style={{alignItems: 'baseline'}}>
-            <RadioButton onChange={
+            <RadioButton data-test-id='radio-lgbtq-yes' id='identifiesAsLgbtqYes' onChange={
```

Comment:
nit: I believe we use dash? casing for most of our HTML id/class names.

Reply:
Yep – fixed.